cocos/physics/cannon/cannon-world.ts

```diff
@@ -76,6 +76,36 @@ export class CannonWorld implements IPhysicsWorld {
         this._world.defaultContactMaterial.frictionEquationRelaxation = 3;
     }
 
+    sweepBox (worldRay: geometry.Ray, halfExtent: IVec3Like, orientation: IQuatLike,
+        options: IRaycastOptions, pool: RecyclePool<PhysicsRayResult>, results: PhysicsRayResult[]): boolean {
+        throw new Error('sweepBox not support in cannon.js.');
+    }
+
+    sweepBoxClosest (worldRay: geometry.Ray, halfExtent: IVec3Like, orientation: IQuatLike,
+        options: IRaycastOptions, result: PhysicsRayResult): boolean {
+        throw new Error('sweepBoxClosest not support in cannon.js.');
```

Comment:
should use EngineErrorMap to reduce code size.

Reply:
ok